.github/actions/spell-check/patterns.txt

```diff
@@ -32,3 +32,12 @@ TestCase\("[^"]+"
 \\testapp
 \\tests
 \\tools
+
+# plugin.json
+^  "ID": "[0-9A-F]{32}",$
+
+# UnitTests
+\[DataRow\(.*\)\]
+
+# Id info inside markdown file (registry.md) 
+^\s+\|\s+ID\s+\|\s*\`[0-9A-F]{32}\`
```

Comment:
Sorry, turns out there was _no_ whitespace at the beginning of the line (I wrote that instruction from a phone and guessed wrong about the markup):
```suggestion
^\|\s+ID\s+\|\s*\`[0-9A-F]{32}\`
```